yolox/core/trainer.py

```diff
@@ -334,3 +360,6 @@ def save_ckpt(self, ckpt_name, update_best_ckpt=False):
                 self.file_name,
                 ckpt_name,
             )
+
+            if self.args.logger == 'wandb':
```

Comment:
please use `'` instead of `"` for normal string.

Reply:
Fixed